lib/hound/internal_helpers.ex

```diff
@@ -79,15 +79,7 @@ defmodule Hound.InternalHelpers do
       ]
     ]
 
-    if driver_info[:driver] == "phantomjs" do
-      if Enum.member?(unsupported_features[:phantomjs], feature) do
-        false
-      else
-        true
-      end
-    else
-      true
-    end
+    driver_info[:driver] == "phantomjs" && !Enum.member?(unsupported_features[:phantomjs], feature)
```

Comment:
We are probably missing some tests, but this should fail.

It is supposed to succeed when `driver_info[:driver] != "phantomjs"`, but with your code it does the opposite.
The correct way to write this would be

```elixir
not (driver_info[:driver] == "phantomjs" and Enum.member?(unsupported_features[:phantomjs], feature))
```

but if we refactor this, maybe we could stop treating PhantomJS differently:
add all the drivers to `unsupported_features` and change the condition to something like

```elixir
not Enum.member?(unsupported_features[driver], feature)
```

Also, we could maybe make `unsupported_features` a module attribute instead of a local variable.

Thank you!